HW3.py

```diff
@@ -0,0 +1,87 @@
+import sys
+import tempfile
+
+from pyspark.sql import SparkSession
+
+import requests
+from pyspark import StorageLevel
+from pyspark.ml.classification import RandomForestClassifier
+from pyspark.ml.feature import CountVectorizer
+
+
+
+def main():
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+    # Load Data directly from MariaDB
+
+    t_rolling_lookup_df = spark.read \
```

Comment:
For some reason, I got an error when running this command at line 18. "_py4j.protocol.Py4JJavaError: An error occurred while calling o32.load.
: java.lang.ClassNotFoundException: org.mariadb.jdbc.Driver_". It must be some Java setup that I don't have in my computer.

Reply:
Yeah but then after that, I can't see if everything else worked.